### PR TITLE
feat: Initial commit

### PR DESCRIPTION
**Pull request by lfvbarcus**
Adds Demo project with minimal examples of reqstool configuration.

### CODE REVIEW

**Review by lfvjimisola — DISMISSED** [review]

File: docs/antora-playbook.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/asciidoctor/asciidoctor-intellij-plugin/main/src/main/resources/jsonSchemas/antoraPlaybookSchema.json
+
+site:
+  title: Reqstool Demo Documentation
```
Comment (lfvjimisola) [inline docs/antora-playbook.yml:4]: Just Reqstool Demo? (it's not a Demo of Documentation)

File: docs/reqstool/ext-001/requirements.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Luftfartsverket/reqstool-client/main/src/reqstool/resources/schemas/v1/requirements.schema.json
+
+metadata:
+  urn: ext-001
+  variant: external
+  title: Some system Requirement Title
```
Comment (lfvjimisola) [inline docs/reqstool/ext-001/requirements.yml:6]: Some System Requirement...

File: docs/reqstool/reqstool_config.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Luftfartsverket/reqstool-client/main/src/reqstool/resources/schemas/latest/requirements-config.schema.json
+
+type: java-maven # custom|java-maven|java-maven-docs|python-pyproject
```
Comment (lfvjimisola) [inline docs/reqstool/reqstool_config.yml:3]: Did we keep the java-maven-docs?

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>se.lfv.reqstool</groupId>
+	<artifactId>reqstool-demo</artifactId>
+	<version>0.0.1</version>
```
Comment (lfvjimisola) [inline pom.xml:8]: We should bump the version so that it is unique (can't recall if we used 0.0.1 internally or not).

Comment (lfvbarcus) [reply pom.xml:8]: yes

File: docs/reqstool/reqstool_config.yml (hunk 3 above)
Comment (lfvbarcus) [reply docs/reqstool/reqstool_config.yml:3]: no, that's gone. Good catch

File: docs/reqstool/ext-001/requirements.yml (hunk 2 above)
Comment (lfvbarcus) [reply docs/reqstool/ext-001/requirements.yml:6]: ok

File: docs/antora-playbook.yml (hunk 1 above)
Comment (lfvbarcus) [reply docs/antora-playbook.yml:4]: Ok

**Review by lfvjimisola — APPROVED** [review]

**Pull request merged**